python/adbc_driver_manager/adbc_driver_manager/dbapi.py

```diff
@@ -669,9 +674,7 @@ def close(self):
         if self._closed:
             return
 
-        if self._results is not None:
-            self._results.close()
-            self._results = None
+        self._clear()
```

Comment:
Should _clear just imply close and closed=true?

Reply:
Er, no, because this object can be reused and _clear is used below, so the only thing it should do is free any allocated array stream

Reply:
Gotcha. Then all good!